js/controllers/wizard/WizardHandler.js

```diff
@@ -103,6 +103,30 @@
 
 
 	//
+	// Private methods
+	//
+	/**
+	 * Show the loading spinner
+	 *
+	 * @return null
+	 */
+	$.pkp.controllers.wizard.WizardHandler.prototype.showProgressIndicator_ =
+			function() {
+		this.getProgressIndicator().css('opacity', 1);
+	};
+
+	/**
+	 * Hide the loading spinner
+	 *
+	 * @return null
```

Comment:
This needs to be `@private` or the linting will fail. I'd suggest creating an OJS branch with just a submodule commit in it, pointing to your branch of pkp-lib using the commit message, in order to get the JS linting going.